cypress/integration/search/post_search_display.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 3]*/
+
+describe('Post search display', () => {
+    it('M14252 After clearing search query, search options display', () => {
+        // 1. Login and navigate to the app
```

Comment:
We're migrating away from adding number for test steps.  Instead, we replaced it with `#`.  Please update.
```suggestion
        // # Login and navigate to the app
```